common/types/src/main/java/io/helidon/common/types/TypeNameSupport.java

```diff
@@ -330,6 +338,38 @@ static TypeName createFromGenericDeclaration(String genericAliasTypeName) {
                 .build();
     }
 
+    private static TypeName createExtends(String typeNames) {
+        // T extends io.helidon.webserver.spi.ServerFeature
+        int index = typeNames.indexOf(" extends ");
+
+        var builder = TypeName.builder()
+                .generic(true)
+                .className(typeNames.substring(0, index));
+        String theOtherPart = typeNames.substring(index + 9);
+        if (theOtherPart.contains(".")) {
+            builder.addUpperBound(TypeName.create(theOtherPart));
+        } else {
+            builder.addUpperBound(TypeName.createFromGenericDeclaration(theOtherPart));
+        }
+        return builder.build();
+    }
+
+    private static TypeName createSuper(String typeNames) {
+        // T extends io.helidon.webserver.spi.ServerFeature
```

Comment:
This says "extends", but then the code uses `" super "`. Copy-and-paste bug?

Reply:
Fixed, luckily only the comment was copied, not the code.